Interface/AddOns/oUF_NeavRaid/tags.lua

```diff
@@ -45,5 +46,6 @@ end
 oUF.Tags.Events['name:raid'] = 'UNIT_NAME_UPDATE'
 oUF.Tags.Methods['name:raid'] = function(unit)
     local name = UnitName(unit)
+    if (name == nil) then return '' end
```

Comment:
For some reason if you try to run oUF_NeavRaid without oUF_Neav running the tags break. When I was trying to fix it I had the function print out the unit and name into chat and it was listing a raid unit with no name. Using this code prevented it from showing any errors and the frames work normally.